dapp/yarn.lock

```diff
@@ -1403,6 +1403,21 @@
   resolved "https://registry.yarnpkg.com/@emotion/unitless/-/unitless-0.7.5.tgz#77211291c1900a700b8a78cfafda3160d76949ed"
   integrity sha512-OWORNpfjMsSSUBVrRBVGECkhWcULOAJz9ZW8uK9qgxD+87M7jHRcvh/A96XXNhXTLmKcoYSQtBEX7lHMO7YRwg==
 
+"@ethersproject/abi@5.0.7":
```

Comment:
We don't need this new dependency, do we?

Reply:
Removed this from `package.json` but haven't rerun the `yarn install`. Thanks for catching it!